site/src/sphinx/client-retry.rst

```diff
@@ -365,14 +365,33 @@ You might want to use :ref:`client-circuit-breaker` with :api:`RetryingHttpClien
 
 This decorates :api:`CircuitBreakerHttpClient` with :api:`RetryingHttpClient` so that the :api:`CircuitBreaker`
 judges every request and retried request as successful or failed. If the failure rate exceeds a certain
-threshold, it raises a :api:`FailFastException`.
-
-You have to be very cautious implementing your own :api:`RetryStrategy` in this situation.
-When a :api:`FailFastException` is raised, you can retry or you shouldn't retry depending on
-you are using :ref:`client-service-discovery` or not. If you are doing client-side load balancing,
-you might want to retry because there's a chance that only small amount of the backends are unreachable
-so that you can send the requests to other hosts. On the contrary, if you are sending the requests to only
-one host, you will not want to retry because the circuit is already closed.
+threshold, it raises a :api:`FailFastException`. When using both clients, you need to write a custom
+:api:`RetryStrategy` to handle this exception so that the :api:`RetryingHttpClient` does not attempt
+a retry unnecessarily when the circuit is open, e.g.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.client.circuitbreaker.FailFastException;
+
+    new RetryStrategy() {
+        final Backoff backoff = Backoff.ofDefault();
```

Comment:
nit: How about removing this? Because there's no place using it.

Reply:
Removed~